administration/package.json

```
@@ -117,7 +118,8 @@
     "format": "NODE_ENV=test eslint . --fix && prettier . --write",
     "generate-graphql": "graphql-codegen --config graphql-codegen.yml",
     "generate-protobuf": "protoc -I ../specs --es_out=src/generated --es_opt=target=ts --plugin=protoc-gen-es=../node_modules/.bin/protoc-gen-es ../specs/card.proto",
-    "postinstall": "npm run generate-protobuf && npm run generate-graphql"
+    "postinstall": "npm run generate-protobuf && npm run generate-graphql",
+    "check-circular-deps": "bash ./scripts/check_circular_deps.sh"
```

:wrench: I think it would be better not to request bash explicitly.
```suggestion
    "check-circular-deps": "./scripts/check_circular_deps"
```
Perhaps you have to run `chmod +x check_circular_deps` first though.

---

Hm so everyone who wants to run in locally has to execute chmod first?
Don't you think its getting more complicated? I mean who doesn't have bash and also our ci has bash

---

how i can do this?

---

just run `chmod +x` and commit the changed file :)

---

hm the file is not recognized as changed @steffenkleinle

---

hm looks like madge then has to be installed globally if its not used with bash https://app.circleci.com/pipelines/github/digitalfabrik/entitlementcard/7403/workflows/69df5df7-5bc9-439f-a457-12ed508e7978/jobs/50765?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary

---

okay i reverted that bash removal. feel free to improve that in a separate pr